.github/workflows/clang.yml

```diff
@@ -0,0 +1,95 @@
+name: Build clang
+# idea - only run on tagged builds?
+# also: opam installs use versions from infer v0.17.0
+# https://github.com/facebook/infer/blob/v0.17.0/opam.locked
+on: [push, pull_request]
```

Comment:
Maybe limit to pushes? Not sure PRs are worth building clang over, especially if we don't run any tests (besides the fact that clang still builds).

Reply:
Sure, that makes more sense.